### PR TITLE
Add Sudoku win-a-tshirt problem

### PR DESCRIPTION
**Pull request by zapme**
Added description and sample files to win a t-shirt.

### CODE REVIEW

File: week-06/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+Special Challenge to win a GitHub t-shirt voucher
+
+[Sudoku Solution Checker](../problems/sudokuchecker)
+
+---
+
+For instructions on the format of ChallengeSet and how to contribute, see the [first weeks `README.md`](../week-01).
+
+Submissions can be submitted past **Friday @ 5:00pm (February 14th, 2020)**.
```
Comment (jackharrhy) [inline week-06/README.md]:
```suggestion
Submissions can be submitted past **Friday @ 5:00pm (February 21st, 2020)**.
```

File: week-06/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Week 5 (Feb 10th)
```
Comment (jackharrhy) [inline week-06/README.md]:
```suggestion
# Week 5 (Feb 17th)
```

**Review by jackharrhy — CHANGES_REQUESTED** [review]
thanks for the problem / giving the t-shirt codes away!

two small changes, both regarding dates

this PR shall be merged tomorrow as per previous problem set releases :)

File: week-06/README.md (hunk 2 above)
Comment (nint8835) [reply week-06/README.md]: @jackharrhy Do you mean:
```suggestion
# Week 6 (Feb 17th)
```

Comment (jackharrhy) [reply week-06/README.md]: _you right_

**Review by jackharrhy — APPROVED** [review]

**Comment by jackharrhy** [comment]
merged with errors, will be fixed in future PR

**Pull request merged**